CHANGELOG.asciidoc

```diff
@@ -87,6 +87,8 @@ https://github.com/elastic/beats/compare/v5.1.2...v5.2.0[View commits]
 - Fix service times-out at startup. {pull}3056[3056]
 - Kafka module case sensitive host name matching. {pull}3193[3193]
 - Fix interface conversion panic in couchbase module {pull}3272[3272]
+- Fix overwriting explicit empty config sections {issue}2918[2918]
+- Fix go routine leak in docker module. {pull}3492[3492]
```

Comment:
Can you cleanup the Changelog in this PR, please? It looks like the cherry-pick added two lines and to the 5.2.0 header. I'm afraid that if we'll merge it like this, we're going to forget to fix one of these two issues.